noxfile.py

```diff
@@ -91,6 +93,50 @@ def cover(session):
     session.run("coverage", "erase")
 
 
+@nox.session(name="emulator-system", python=ALL_INTERPRETERS)
+def emulator_system(session):
+    """Run the system test suite."""
+    # Only run the emulator tests manually.
```

Comment:
I think it would be good to run this in CI (once all tests are consistently passing)

Reply:
I agree.  However, there are ~30 tests that aren't passing yet.